README.md

```diff
@@ -4,7 +4,7 @@
 
 ### License ###
 
-**hashcat** is licensed under the MIT license. Refer to [docs/license.txt](docs/license.txt) for more information.
+**hashcat** is licensed under the MIT license. Refer to License tab for more information.
```

Comment:
License tab is GitHub feature, when someone download source code and open the `README.md` there are no tabs.